content/es/blog/2021-11-02-good-whale’s-grant-fund.md

```diff
@@ -0,0 +1,34 @@
+---
+title: La subvención de la ballena buena
+date: 2021-11-02T12:02:45.749Z
+author: Ergo Foundation (translated by Daniu, original version will always prevail)
+authorPhoto: /img/uploads/1762.png
+blogPhoto: /img/uploads/good-whale-cover-1-.png
+tags:
+  - Ergo Team
+  - Building Ergo
+---
+<!--StartFragment-->
+
+Ergo cree que las personas deben tomar tanto control del ecosistema posible para apoyar la descentralización y las esperanzas de desarrollar la comunidad criptográfica y el ecosistema a través de los movimientos de base.El ecosistema ERGO está creciendo a un ritmo rápido en términos de desarrollo y base de usuarios.Construimos en colaboración gracias a todos los ergonautas que apoyan y toman el control del ecosistema.
+
+Alex (Kushti) Chepurnoy ha publicado recientemente (https://www.ergoforum.org/t/good-whes-grant-fund/2779) una proposición de un gran inversor de ergo llamado * Buena ballena *.En su deseo de apoyar la investigación y el desarrollo de Ergo, la buena ballena está ofreciendo subvenciones de hasta $ 20,000 SIGUSD (¡cada uno!) este no es un trabajo por contrato, por lo que los desarrolladores que solicitan esta subvención también controlarán los productos creados y cualquier flujo de ingresos resultante.
+
+La ballena buena está interesado principalmente en los siguientes temas:
+
+* Desarrollar un estándar para certificados basados en Nipopow para NFTS
+* Desarrollo de contratos y UI para el Ergo Name System (alternativa para Namecoin, HandShake, Ethereum Name Service)
+* Desarrollar aplicaciones en la parte superior de Ergodex, como DEXYUSD [DEXY USD - STABLECOIN DISEÑO SIMPLEST] (https://www.eroforum.org/t/dexy-usd-simplest-stablecoin-design/1430), Ventas de tokens con garantías de recompra[NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo] (https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/), etc.
```

Comment:
need to merge [NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo] (https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/)
 to [NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo](https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/)